### ISSUE TITLE
Add new Machine Learning projects to make learning easy

### ISSUE THREAD

**Issue by itsdivya1309** [itsdivya1309/ML_Algos#1]
I would like some help from the open source community in making this repository a holistic resource for beginners to intermediate ML enthusiasts. The main aim is to make this a one-stop solution for people looking for ML projects so that they can learn with hands-on-tutorials.

**Comment by AquarlisPrime** [comment]
can add gradient descent and XGBRegressor too along with, tensorflow intro here

**Comment by itsdivya1309** [comment]
Would like it if you could open a pull request

**Comment by AquarlisPrime** [comment]
Yeah, I can. Will do it as soon as possible.

Regards,
Anjali Singh
